larq/quantized_variable_test.py

```diff
@@ -0,0 +1,279 @@
+import pytest
+import tensorflow as tf
+from numpy.testing import assert_almost_equal, assert_array_equal
+from tensorflow.python.distribute.values import DistributedVariable
+
+from larq import quantized_scope
+from larq.quantized_variable import QuantizedVariable, create_quantized_variable
+from larq.testing_utils import evaluate
+
+
+def get_var(val, dtype=None, name=None):
+    return tf.compat.v1.Variable(val, use_resource=True, dtype=dtype, name=name)
+
+
+@pytest.fixture(params=[True, False])
+def quantized(request):
+    """pytest fixture for running test quantized and non-quantized"""
+    with quantized_scope.scope(request.param):
+        yield request.param
```

Comment:
I think using strings instead of bools would be nicer here too.

Reply:
And maybe we can move this into `conftest.py`, where most other pytest fixtures are. (There's only one other fixture that's not in that file: one in `models_test.py`; I think moving them all to `conftest.py` to have one place to look for fixtures would make sense.)

Reply:
I think I like returning booleans since it makes using the fixture in the test very easy since I only have to do `if quantized:` and not check if the return value matches a string. I put it here since this is the only file we need to use it.

Reply:
OK booleans are good with me.

For the fixture location: we also have the `distribute_scope` and `eager_and_graph_mode` fixtures in `conftest.py` even though they only get used by `quantized_variable_test.py`, so I think we should treat the `quantized` fixture the same. We could then also test it in `conftest_test.py` like the other two.